modules/broker-amqp/src/main/java/io/ballerina/messaging/broker/amqp/AmqpConsumer.java

```diff
@@ -122,7 +123,7 @@ private ErrorLogger(String queueName) {
         @Override
         public void operationComplete(ChannelFuture future) {
             if (!future.isSuccess()) {
-                LOGGER.warn("Error while sending message for " + queueName, future.cause());
+                LOGGER.warn("Error while sending message for {}", queueName, future.cause());
```

Comment:
Does this result in getting the stack trace trace printed? The reason I am telling this is that this result in calling  `void warn(String var1, Object var2, Object var3);` method in the Logger interface. Maybe they do some magic inside the implementation of the Logger.

Reply:
Yep, they are handling the stack-trace internally